src/bindgen.rs

```diff
@@ -19,7 +19,7 @@ pub fn wasm_bindgen_build(
     disable_dts: bool,
     target: Target,
     profile: BuildProfile,
-) -> Result<(), failure::Error> {
```

Comment:
This has changed in the main branch, need to add `extra_options`.:
```rust
    extra_options: &Vec<String>,
) -> Result<()> {
```

Reply:
fixed